src/cli.ts

```diff
@@ -8,7 +8,7 @@ import {
   calculateMeasureReports,
   calculateGapsInCare,
   calculateRaw,
-  calculateDataRequirements,
+  calculateMeasureDataRequirements,
```

Comment:
Are we sure we want to change the name of this right now? That would make this a breaking change potentially or am I wrong? If we do indeed want to change it, we should probably update the README.

Reply:
Oh- good call. I will revert this.